x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/expression/function/scalar/multivalue/MvMax.java

```diff
@@ -69,18 +68,7 @@ public String getWriteableName() {
 
     @Override
     protected TypeResolution resolveFieldType() {
-        return isType(
-            field(),
-            t -> isSpatial(t) == false && isRepresentable(t),
-            sourceText(),
-            null,
-            "boolean",
-            "date",
-            "ip",
-            "string",
-            "version",
-            "numeric except counter types"
-        );
+        return isRepresentableExceptCountersAndSpatial(field(), sourceText(), DEFAULT);
```

Comment:
I'm changing this list of most types to "any but counters and spatial", which should be enough here